nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -71,6 +71,7 @@
     public static final String ADMINISTRATIVE_YIELD_DURATION = "nifi.administrative.yield.duration";
     public static final String PERSISTENT_STATE_DIRECTORY = "nifi.persistent.state.directory";
     public static final String BORED_YIELD_DURATION = "nifi.bored.yield.duration";
+    public static final String PROCESSOR_START_TIMEOUT = "nifi.processor.start.timeout";
```

Comment:
In looking over this PR more, I'm quite puzzled by this property - it does not appear to be read anywhere either. Is this ever used?